lib/inbox.rb

```diff
@@ -84,6 +83,14 @@ def self.interpret_response(result, result_content, options = {})
 
 
   class API
+    if RUBY_PLATFORM[/java/] == 'java'
+      require 'nylas/inbox/java'
+      include Nylas::Inbox::Java
```

Comment:
I like the idea of having platform-specific modules, but I'd prefer another name here. What about `Nylas::Java`?